lib/node.js

```diff
@@ -109,6 +109,10 @@ class Node {
       nameFromFolder(path || pkg.name || realpath) ||
       pkg.name
 
+    // set a manual name if running from fs root dir with no package.json
+    if (!this.name && path === '/')
+      this.name = 'system-root'
```

Comment:
what happens if i have a dir like `/system-root`, or a package named "system-root"?

Reply:
then `this.name` should already be a truthy value (`system-root`) and it would never reach this loc

Reply:
Sure, but is there an issue that two very different paths would result in the same name?

Reply:
yeah, it might not be an issue or maybe it is but either way you bring up good points and we decided it's better to wait till next week and have some feedback from @isaacs on it 😊